Meet Up - JavaScript/README.md

```diff
@@ -0,0 +1,15 @@
+# chrome-extension-meetup
+Its a simple extension which can open and store all the added meeting links to join quickly.
```

Comment:
There are some grammatical mistakes present, please correct them.

Reply:
Yes I have updated the README